Straight_skeleton_2/doc/Straight_skeleton_2/CGAL/extrude_skeleton.h

```diff
@@ -0,0 +1,94 @@
+namespace CGAL {
+
+/**
+* \ingroup PkgStraightSkeleton2Extrusion
+*
+* \brief constructs the straight skeleton-based extrusion of a polygon with holes.
+*
+* Given a polygon with holes and a set of weights, the skeleton extrusion is a volume constructed
+* from the weighted straight skeleton by associating a height to the vertices of the skeleton,
+* which corresponds to the time at the vertex. The input polygon is placed at `z = 0`.
+*
+* This function allows cropping the extruded skeleton at a maximum height, using the optional
+* `maximum_height` named parameter.
+*
+* The result is a closed, 2-manifold surface triangle mesh. Note that this mesh can have non-local
```

Comment:
is the triangulation optional?

Reply:
Right now it is not, but it could be, adding to @todo, I'll do it if I have time.